### PR TITLE
Feat/novedades/extender servicio http base novedades

### PR DESCRIPTION
**Pull request by MartinCaparros**
UserStory: https://alkemy-labs.atlassian.net/browse/OT91-76

### CODE REVIEW

**Comment by saralavanchy** [comment]
Please add a description and add a screenshot of what is being done here.

File: src/Services/newsServices.js [hunk 1]
```diff
@@ -23,6 +28,16 @@ const createOrUpdateNews = async (news) => {
...
+  try {
+    await axios.delete(`http://ongapi.alkemy.org/public/api/news/${newToRemove.categoryId}`)
+  }catch(error){
+    console.log(error)
+  }
+}
+
 export const createNewsObject = (id, name, image, createdAt) => ({id, name, image, createdAt});
 
 export { createOrUpdateNews };
```
Comment (saralavanchy) [inline src/Services/newsServices.js]: use this export to export the functions, add all the methods:
```
{ createOrUpdateNews, deleteNewsById, getNews, getNewById }
```

Comment (saralavanchy) [reply src/Services/newsServices.js]: The create and update actions should have a individual function

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Services/newsServices.js (hunk 1 above)
Comment (MartinCaparros) [reply src/Services/newsServices.js]: Perfect Sara, already fixed. Thx

**Comment by MartinCaparros** [comment]
I leave you here the missing evidence screenshot.

News Services functional

**Review by saralavanchy — APPROVED** [review]

File: src/Services/newsServices.js [hunk 2]
```diff
@@ -6,23 +6,54 @@ const getNews = async () => {
   return data;
 };
 
+const getNewById = async (id) => {
+    const {data}= await axios.get(`http://ongapi.alkemy.org/api/news/${id}`)
+    return data
+}
+
+const updateNewById = async (news) => {
```
Comment (saralavanchy) [inline src/Services/newsServices.js:14]: you should send the new to update here

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Services/newsServices.js (hunk 2 above)
Comment (MartinCaparros) [reply src/Services/newsServices.js:14]: Solved! Thx

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**